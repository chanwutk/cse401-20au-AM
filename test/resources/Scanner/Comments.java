/* Here are some comments */
some code
// Here are some line comments
some code /*
Here are
some block comments */ more code