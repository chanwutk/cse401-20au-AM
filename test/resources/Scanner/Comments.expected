ID(some) ID(code) ID(some) ID(code) ID(more) ID(code)